### PR TITLE
Update TOC and Landing Page with Model Builder tutorial

### PR DESCRIPTION
**Pull request by luisquintanilla**
[Landing Page Preview Link](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/index?branch=pr-en-us-15534)

[AB#1626493](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1626493)

### CODE REVIEW

**Comment by luisquintanilla** [comment]
Retriggering Build

File: docs/machine-learning/toc.yml [hunk 1]
```diff
@@ -41,12 +41,14 @@
       href: tutorials/text-classification-tf.md
   - name: Tools
```
Comment (cjgronlund) [inline docs/machine-learning/toc.yml]:
```suggestion
  - name: Model Builder & CLI
```

File: docs/machine-learning/index.yml [hunk 2]
```diff
@@ -56,7 +56,9 @@ landingContent:
           - text: Analyze website comment sentiment (Model Builder)
             url: tutorials/sentiment-analysis-model-builder.md   
           - text: Predict prices (Model Builder)
-            url: tutorials/predict-prices-with-model-builder.md                 
+            url: tutorials/predict-prices-with-model-builder.md     
+          - text: Categorize health violations (Model Builder)
```
Comment (cjgronlund) [inline docs/machine-learning/index.yml]:
```suggestion
          - text: Categorize health violations (Model Builder & SQL Server)
```

File: docs/machine-learning/toc.yml [hunk 3]
```diff
@@ -41,12 +41,14 @@
...
     items:
-    - name: Predict prices with Model Builder
+    - name: Predict prices with Model Builder (regression)
```
Comment (cjgronlund) [inline docs/machine-learning/toc.yml]:
```suggestion
    - name: Predict prices (Model Builder & regression)
```

File: docs/machine-learning/toc.yml [hunk 4]
```diff
@@ -41,12 +41,14 @@
...
       href: tutorials/predict-prices-with-model-builder.md
     - name: Analyze sentiment using the ML.NET CLI
       href: tutorials/mlnet-cli.md
-    - name: Analyze sentiment using Model Builder and Razor Pages
+    - name: Analyze sentiment using Model Builder and Razor Pages (binary classification)
```
Comment (cjgronlund) [inline docs/machine-learning/toc.yml]:
```suggestion
    - name: Analyze sentiment in Razor Pages (Model Builder & binary classification)
```

File: docs/machine-learning/toc.yml [hunk 5]
```diff
@@ -41,12 +41,14 @@
...
       href: tutorials/sentiment-analysis-model-builder.md
+    - name: Categorize health violations with Model Builder (multiclass classification)
```
Comment (cjgronlund) [inline docs/machine-learning/toc.yml]:
```suggestion
    - name: Categorize health violations (Model Builder & multiclass classification)
```

**Review by cjgronlund — COMMENTED** [review]
:>)

**Review by cjgronlund — APPROVED** [review]

**Pull request merged**